Improve the quick start guide to make the start order more clear. (#115)
UniversalRobots/Universal_Robots_ROS_Driver: README.md

[package's documentation](ur_calibration/README.md) for details.

#### Quick start
Once the driver is built and the **externalcontrol** URCap is installed on the robot, you are good

> to go ahead starting the driver. (**Note**: We do recommend, though, to calibrate your robot first.)

Reviewer: Nit, but: we're not actually calibrating the robot, are we? It's more exporting the already present calibration data and converting it to be usable with ROS tools.

Author: true...

Author: Fixed in 8c86b7e

For more information on the launch file's parameters see its own documentation.

> Once the robot driver is started, load the previously generated program on the robot panel that will

Reviewer: Which program are we referring to here?

> start the `External Control` programnode and execute it. From that moment on the robot is fully

Reviewer: ```suggestion
start the *External Control* program node and execute it. From that moment on the robot is fully
```

> functional. You can make use of the pause function or even stop the program. Simply press the play

Reviewer: ```suggestion
functional. You can make use of the *Pause* function (▶) or even *Stop* the program (◾). Simply press the *Play*
```

> button again and the ROS driver will reconnect.

Reviewer: ```suggestion
button (▶) again and the ROS driver will reconnect.
```

> Inside the ROS terminal running the driver you should see the output `Robot ready to receive control commands.`

Reviewer: ```suggestion
Inside the ROS terminal running the driver you should see the output *Robot ready to receive control commands*.
```

Author: I decided against that, as we are referring to actual terminal output here.
